ccd-definition/ComplexTypes/RepaymentPlanLRspec-prod.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "ID": "RepaymentPlanLRspec",
```

Comment:
are we sure we want this in production?

Reply:
Or have you just duplicated so that we are defining prod and non prod?

Reply:
I just duplicated into prod and non-prod, then remove the line in CUI-nonprod

Reply:
Having a discussion with both and the change with be in base file as it is not impact the functionality. This file will be removed